history/taskana-simplehistory-provider/src/test/java/acceptance/AbstractAccTest.java

```diff
@@ -195,156 +141,38 @@ protected ObjectReference createObjectRef(
   }
 
   protected static WorkbasketHistoryEventMapper getWorkbasketHistoryEventMapper() {
-
-    SqlSessionManager manager = null;
-
-    Field sessionManager;
     try {
-      sessionManager = TaskanaHistoryEngineImpl.class.getDeclaredField("sessionManager");
-
+      Field sessionManager = TaskanaHistoryEngineImpl.class.getDeclaredField("sessionManager");
```

Comment:
This is our common way to do this because we cannot access it via the public API?

Reply:
I didn't ask this myself. All I did here is clean up the method.
Typically our acceptance tests should not access any internal. The redesign of our ArchitectureTests should enforce that rule.
Until then I wouldn't change it. This is out of the scope of this ticket.